docs/Coding/internationalization.rst

```diff
@@ -0,0 +1,170 @@
+..
+    SPDX-License-Identifier: GPL-3.0-or-later
+    SPDX-FileCopyrightText: 1996-2021 Freeciv Contributors
+    SPDX-FileCopyrightText: 2022 James Robertson <jwrober@gmail.com>
+    SPDX-FileCopyrightText: 2022 Louis Moureaux <m_louis30@yahoo.com>
+
+Internationalization
+********************
+
+Internationalization, or i18n for short, refers to making sure software is ready to be adapted for use in a
+language and country other than the original. Freeciv21 uses American English as the primary target language,
```

Comment:
It is technically "US English", maybe change to American (US) English?

Reply:
Both are correct, I added the ISO code